### DISCUSSION TITLE
Is it possible to disable worktree creation for `werf build`?

### DISCUSSION THREAD

**Discussion by slonopotamus** [werf/werf#4760] [General]
I have multi-GB repos (50-300GB). `werf build` in such thing involves creation of worktree that takes up huge amounts of disk space. Is it possible to tell `werf build` to use my existing working copy instead of creating a new one under `~/.werf/local_cache/git_worktrees`?

**Comment by alexey-igrychev** [comment]
Currently, the working tree is used to prepare the build context, and there is no alternative path.
